lib/src/main/java/com/thenewboston/utils/Pagination.kt

```diff
@@ -0,0 +1,33 @@
+package com.thenewboston.utils
+
+data class PaginationOptions(
+    val offset: Int,
+    val limit: Int
+) {
+    fun toQuery() = "?offset=$offset&limit=$limit"
+}
+
+object PAGE {
+    var DEFAULT: PaginationOptions = PaginationOptions(
+        offset = 0,
+        limit = 20
+    )
+
+    var THIRTY_ITEMS: PaginationOptions = PaginationOptions(
+        offset = 0,
+        limit = 10
+    )
+}
+
+data class PaginationResult<T> (
+    val limit: Int,
+    val item: T
+) {
+
+    fun toList(): List<T> {
+        val list: ArrayList<T> = arrayListOf()
+        for (i in 1..limit) list.add(item)
+
+        return list
+    }
```

Comment:
This class is only used in test classes. Can we move it to the test package? if we don't, it will count as uncovered test code.

Reply:
done